### ISSUE TITLE
Rapid Response: URL for Open Data Day website

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/mofo-devops#461]
The Advocacy team is working on a quick build for a petition site to align with Open Data Day on March 4th. The site will be deployed on Friday, March 3.

Desired url options coming soon from @shaghdoosti 

@cadecairos - do you think this timeline is feasible? Please let us know about any potential blockers.

**Comment by cadecairos** [comment]
Make sure to note URL options in the mofo-devops-private repo.

I can't comment on the timeline without knowing some of the technical details of the petition: 

- what's it build with
- how are we hosting it
- what kind of integrations do we need

**Comment by hannahkane** [comment]
@ScottDowne can probably say more, but this will be the exact same setup as the Copyright campaign site we're building

**Comment by shaghdoosti** [comment]
And I just emailed @ScottDowne and @cadecairos the URL options as well.

**Comment by ScottDowne** [comment]
@cadecairos This is a copy of the copyright, with new copy. It's also more of a petition, and less of a signup. So it'll work more like our old copyright from a salesforce pov. Does that make sense?

**Comment by cadecairos** [comment]
Just to update:

We have purchased the domain (`iheartopendata.org`), and can configure it when we're ready. I'll get the domain approved in Digicert next week (It only takes a couple minutes, and I can do it without help from WebOps now).

I've set up a Heroku pipeline with two apps.
pipelinename: iheartopendata-org
staging app: iheartopendata-org-staging
production app: iheartopendata-org-production
